src/libs/bin_decoder/lv_bin_decoder.c

```diff
@@ -528,6 +529,87 @@ static lv_result_t decode_indexed(lv_image_decoder_t * decoder, lv_image_decoder
 #endif
 }
 
+static lv_result_t load_indexed(lv_image_decoder_t * decoder, lv_image_decoder_dsc_t * dsc)
+{
+#if LV_BIN_DECODER_RAM_LOAD == 0
+    LV_UNUSED(decoder); /*Unused*/
+    LV_UNUSED(dsc); /*Unused*/
+    LV_LOG_ERROR("LV_BIN_DECODER_RAM_LOAD is disabled");
+    return LV_RESULT_INVALID;
+#else
+
+    LV_UNUSED(decoder); /*Unused*/
+
+    lv_result_t res;
+    uint32_t rn;
+    decoder_data_t * decoder_data = dsc->user_data;
+
+    if(dsc->header.flags & LV_IMAGE_FLAGS_COMPRESSED) {
+        /*The decompressed image is already loaded to RAM*/
```

Comment:
Why does "if the image is compressed" mean "it's already loaded to RAM"?

Reply:
The comment should say "If the image is compressed, then it's already uncompressed to RAM at this stage." Should I modify the comment?

Reply:
I see. I'm just trying to catch up wit the new decoder code :slightly_smiling_face: 

No modification needed.